Assets/Scripts/Items/Editor/WorldItemPropertyDrawer.cs

```diff
@@ -7,15 +7,18 @@
 [CustomEditor(typeof(WorldItem))]
```

Comment:
So I've realized that, contrary to my previous advice, you don't actually need to mess with the property drawer here. You can just make the isAnimate field viewable in the inspector (remove the [HideInInspector] attribute) and it should handle it appropriately without needing edits to this script.

Sorry for the misleading instructions before!